api/predict.py

```diff
@@ -10,7 +10,8 @@
 
 label_prediction = MAX_API.model('LabelPrediction', {
     'age_estimation': fields.Integer(required=True, description='Estimated age for the face'),
-    'face_box': fields.List(fields.Integer(required=True), description='Bounding box coordinates for the face')
+    'detection_box': fields.List(fields.Float(required=True), description='Bounding box coordinates for the face, ' + \
+         'in the form of an array of normalized coordinates [xmin, ymin, xmax, ymax]. Each coordinate is in the range [0, 1]')
```

Comment:
In the README you said the order is `[ymin, xmin, height, width]`. Which is the correct order?

Reply:
Thanks @bdwyer2. The format is now consistent with our models with [ymin, xmin, height, width].

Reply:
Okay, then you should change [README.md#5](https://github.com/IBM/MAX-Facial-Age-Estimator/pull/28/files#diff-04c6e90faac2675aa89e2176d2eec7d8R5)

Reply:
So you will change this?
```suggestion
         'in the form of an array of normalized coordinates [ymin, xmin, height, width]. Each coordinate is in the range [0, 1]')
```

Reply:
Yes